src/Redux/api.tsx

```diff
@@ -164,10 +170,13 @@ const routes = {
 
   userListSkill: {
     path: "/api/v1/users/{username}/skill/",
+    method: "GET",
+    TRes: Type<PaginatedResponse<SkillModel>>(),
   },
 
   userListFacility: {
     path: "/api/v1/users/{username}/get_facilities/",
+    method: "GET",
     TRes: Type<UserModel>(),
```

Comment:
@thedevildude are you sure you've resolved the merge conflicts correctly? Why would get facilities API return a User?

```suggestion
    TRes: Type<FacilityModel[]>(),
```

Reply:
My bad, I might have got confused with the returned api result